daemon/api_debug_stacktrace.go

```diff
@@ -0,0 +1,33 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package daemon
+
+import (
+	"runtime"
+	"strings"
+)
+
+func getStacktrace() Response {
+	stack := make([]byte, 65535)
+	all := true
+	n := runtime.Stack(stack, all)
+	data := strings.Split(string(stack[:n]), "\n")
```

Comment:
Why split the string into lines?

Reply:
No deep reason / no strong opinion. I'm fine with just returning a text blob.

Reply:
Done.